edx_sga/sga.py

```diff
@@ -348,6 +354,25 @@ def get_student_data():
             'display_name': force_text(self.display_name)
         }
 
+    def get_assignments(self):
```

Comment:
The `sort_assignment_by_date` function and this function should be combined. We can all it `get_sorted_submissions` or something like that. "Assignments" is not the correct term in this context